bannou/database/models/tags.py

```diff
@@ -0,0 +1,101 @@
+from __future__ import annotations
+
+import asyncpg
+import attr
+import hikari
+
+from bannou.database.base import DatabaseModel
+
+
+@attr.define()
```

Comment:
```suggestion
@attr.define(weakref_slot=False)
```
Since we wont be using it and saves on memory